apps/test/unit/templates/lessonOverview/LessonStandardsTest.js

```diff
@@ -37,5 +39,71 @@ describe('LessonStandards', () => {
       expect(text).to.contain(standard.shortcode);
       expect(text).to.contain(standard.description);
     });
+
+    it('renders many standards with all standards expanded', () => {
+      const standards = cspStandards.concat(cstaStandards);
+      const wrapper = mount(
+        <LessonStandards standards={standards} expandMode={ExpandMode.ALL} />
+      );
+
+      const frameworks = wrapper.find('Framework');
+      expect(frameworks.length).to.equal(2);
+      frameworks.forEach(framework => {
+        expect(isOpen(framework)).to.be.false;
```

Comment:
I'm confused. The name of the test says all the standards should be expanded but then this checks that isOpen is false. Is this test case named wrong?

Reply:
whoa. it looks like i accidentally pasted a test case within a test case. I'm assuming this code was not getting run. this is fixed now.

I also realized I wasn't testing the case where there is no parent category, because I was only looking for categories that were within parent categories. this is fixed now too.